torchci/lib/drciUtils.ts

```diff
@@ -3,14 +3,16 @@ export const REPO: string = "pytorch";
 export const OWNER: string = "pytorch";
 export const DRCI_COMMENT_START = "<!-- drci-comment-start -->\n";
 export const OH_URL =
-  "https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours";
+    "https://github.com/pytorch/pytorch/wiki/Dev-Infra-Office-Hours";
 export const DOCS_URL = "https://docs-preview.pytorch.org/";
 export const PYTHON_DOCS_URL = "/index.html";
 export const CPP_DOCS_URL = "/cppdocs/index.html";
 export const DRCI_COMMENT_END = `\n
 This comment was automatically generated by Dr. CI and updates every 15 minutes.
 <!-- drci-comment-end -->`;
-export const POSSIBLE_USERS = ["swang392"];
+export const POSSIBLE_USERS = ["swang392", "janeyx99", "atalman", "clee2000", "seemethere",
```

Comment:
can we move this into rolloutUtils.ts? That we have all of our allowlist logic in the same place.